fe/fe-core/src/main/java/org/apache/doris/datasource/hive/source/HiveScanNode.java

```diff
@@ -446,12 +448,11 @@ protected Map<String, String> getLocationProperties() throws UserException  {
     protected TFileAttributes getFileAttributes() throws UserException {
         TFileTextScanRangeParams textParams = new TFileTextScanRangeParams();
         java.util.Map<String, String> delimiter = hmsTable.getRemoteTable().getSd().getSerdeInfo().getParameters();
-        if (delimiter.containsKey(PROP_FIELD_DELIMITER)) {
-            if (delimiter.get(PROP_FIELD_DELIMITER).length() == 0) {
-                textParams.setColumnSeparator(DEFAULT_FIELD_DELIMITER);
-            } else {
-                textParams.setColumnSeparator(delimiter.get(PROP_FIELD_DELIMITER));
-            }
+        // For column separator, first find "field.delim", then "serialization.format".
+        String columnSeparator = delimiter.getOrDefault(PROP_FIELD_DELIMITER,
+                delimiter.getOrDefault(PROP_SERIALIZATION_FORMAT, ""));
```

Comment:
```suggestion
                delimiter.getOrDefault(PROP_SERIALIZATION_FORMAT, DEFAULT_FIELD_DELIMITER));
```

Reply:
Can not give. Because if we dont find PROP_FIELD_DELIMITER and PROP_SERIALIZATION_FORMAT, we should continue find PROP_SEPARATOR_CHAR